Tech and Fin analysis/TA.py

```diff
@@ -12,12 +12,14 @@ def sma(ticker):
     df[['SMA']].plot(figsize=(12,12))
     # plt.show()
     return plt
+    return plt.show()
 
 def ema(ticker):
     df = yf.Ticker(ticker).history(period='1y')
     # print(df.head())
     df['EMA'] = ta.EMA(df['Close'],20)
     df[['EMA']].plot(figsize=(12,12))
+    a = 'ty'

```

Comment:
**Issue**: Incorrect indentation after control structures

**Reason**: The line 'a = 'ty'' is incorrectly indented with 4 spaces instead of being aligned with the function body.

**Suggestion**: Remove the extra space at the beginning of the line to align it with the function body.